tests/reporter/kubernetes.go

```diff
@@ -1208,6 +1208,7 @@ func (r *KubernetesReporter) executeVirtLauncherCommands(virtCli kubecli.Kubevir
 		{command: bridgeJVlanShow, fileNameSuffix: "brvlan"},
 		{command: bridgeFdb, fileNameSuffix: "brfdb"},
 		{command: "env", fileNameSuffix: "env"},
+		{command: "[ -f /var/run/kubevirt/passt.log ] && cat /var/run/kubevirt/passt.log", fileNameSuffix: "passt"},
```

Comment:
But the logs are reflected to `stdout` due to a goroutine that logs it. Or am I missing something?

Reply:
I do not understand, this is still done here in this PR. You read this file and post it on stdout.
So why do we need to collect this intermediate file?

Reply:
Per the understanding we have now about the inability to echo this log to the pod stdout, I guess it makes sense to keep this now.

Reply:
IMO this make sense to keep in any case so we could debug passt in case something went wrong and have no passt logs recorded in virt-launcher logs.